src/pages/dmt5/index.tsx

```diff
@@ -23,7 +23,12 @@ import DMT5BG2 from 'images/svg/dmt5/dmt5-bg2.svg'
 import { size } from 'themes/device'
 import { isBrowser } from 'common/utility'
 
-const meta_attributes = {
+type MetaAttributesType = {
+    og_title: string
+    og_description: string
+}
+
+const meta_attributes: MetaAttributesType = {
```

Comment:
This can be a common type to be used by all these instances. Could be in `src/types/page.types.ts` possibly.

Reply:
Created folder and file the same as you wrote